puppet/manifests/activemq-development-new.pp

```diff
@@ -0,0 +1,6 @@
+node 'dev-box' {
```

Comment:
Is this file the new version for 'activemq-development.pp'? Should one of the files be deleted?